src/libs/actions/IOU.js

```diff
@@ -388,9 +386,14 @@ function requestMoney(report, amount, currency, payeeEmail, payeeAccountID, part
 
     let isNewReportPreviewAction = false;
     let reportPreviewAction = isNewIOUReport ? null : ReportActionsUtils.getReportPreviewAction(chatReport.reportID, iouReport.reportID);
-    if (!reportPreviewAction) {
+    if (reportPreviewAction) {
+        reportPreviewAction.created = DateUtils.getDBTime();
+        const message = ReportUtils.getReportPreviewMessage(iouReport, reportPreviewAction);
+        reportPreviewAction.message[0].html = message;
+        reportPreviewAction.message[0].text = message;
```

Comment:
NAB we can do this in a follow up, but I see this logic is duplicated in a few places, so maybe we can create a function `updateReportPreviewAction` to do this.

Reply:
+1